core/src/test/scala/com/github/scalalab3/logs/ActorsTest.scala

```diff
@@ -0,0 +1,15 @@
+package com.github.scalalab3.logs.core
+
+import akka.actor.{Props}
+import com.github.scalalab3.logs.tests.AkkaSpec
+
+class SystemActorTest extends AkkaSpec {
+  "Test SystemActor" >> {
+    val actorRef = system.actorOf(Props(classOf[SystemActor]), "system-actor")
```

Comment:
Да, я читал, но там доке даже написано:
[It is highly recommended to stick to traditional behavioural testing (using messaging to ask the Actor to reply with the state you want to run assertions against), instead of using TestActorRef whenever possible.](http://doc.akka.io/docs/akka/snapshot/scala/testing.html#Synchronous_Unit_Testing_with_TestActorRef)

Так что не сильно большая проблема запустить реальный актор, по-крайней мере пока тесты у нас не сильно долго работают